manifests/instance.pp

```diff
@@ -290,8 +294,8 @@
         $tls_config = {
           'xpack.security.transport.ssl.enabled' => true,
           'xpack.security.http.ssl.enabled'      => true,
-          'xpack.ssl.keystore.path'              => $_keystore_path,
-          'xpack.ssl.keystore.password'          => $keystore_password,
+          'xpack.http.ssl.keystore.path'         => $_keystore_path,
```

Comment:
Are these options backwards compatible? Otherwise there should probably be a check on the major version and a switch between these two configurations?

Reply:
We had to add a check for major version on our cluster settings... this may be useful:

```
  $es_major_version = $elasticsearch::version.split(/\./)[0] # 7.5.1 -> 7
```

Reply:
This has been added for ssl section